### PR TITLE
'Add comment' CTA at top of page #352

### PR DESCRIPTION
**Pull request by jpmaHub**
github

### CODE REVIEW

**Comment by jpmaHub** [comment]
**If there is no comments or less than 2 comments.** 

Screenshot 2022-09-21 at 15 54 55

File: futurenhs.app/components/_pageTemplates/GroupDiscussionTemplate/index.tsx [hunk 1]
```diff
@@ -154,6 +154,43 @@ export const GroupDiscussionTemplate: (props: Props) => JSX.Element = ({
...
+    
+
+    const handleSubmitReplyClick = (event: any) => {
+        event.preventDefault()
+
+        const rteInstance: any = (
+            window as any
+        ).tinymce?.get('content')
+
+        if (rteInstance) {
```
[the remark this answers is not in the document]
Comment (jpmaHub) [reply futurenhs.app/components/_pageTemplates/GroupDiscussionTemplate/index.tsx:169]: Screenshot 2022-09-21 at 16 08 02
I get this error if i remove the else statment.

**Comment by jpmaHub** [comment]
@simplybenuk @tim-hunt Hi, how important is to have focus on the comment box once you scroll to the bottom? Depends on the nature of the code written, had to add timer to wait between scroll and focus on the box. the delay between scroll and see the cursor on the box may increase depends on the user devices, internet speed.

**Comment by simplybenuk** [comment]
Accessibility should always take precedent. Is there something we can remove that has introduced this?

**Comment by jpmaHub** [comment]
> @jpmaHub Accessibility should always take precedent. Is there something we can remove that has introduced this?

No, I don't think so. It just nature of how that textbox has created. It is wrapped under nhs textbox wrapper and that makes it harder to add focus on it. Anything else you want to add Rio @rio-k9 ?

**Comment by simplybenuk** [comment]
@jpmaHub "Hi, how important is to have focus on the comment box once you scroll to the bottom?"
I don't think that's important. As long as people can still tab or click to focus, that should be fine.

**Comment by rio-k9** [comment]
Hey @simplybenuk, it's all down the the text box library that was decided on by the previous developer. It's not NextJS compatible so a hacky work around was implemented to facilitate the text box auto-focus. We copied the code and re-implemented it for the reply button, but it only works if everything is in the same viewport. 

The moment the page grows bigger, for example if you have a long comment chain, you have to do a work-around that is a bit inconsistent.

**Comment by jpmaHub** [comment]
Going to close this branch as it is not up to date with the latest main branch. The new PR for this issue will be found here https://github.com/nhsengland/futurenhs/pull/364

**Pull request closed without merging**